templates/kotlin/mobile/app/src/main/kotlin/app_package/utils/biometrics/BiometricManagerV23.kt.ftl

```diff
@@ -0,0 +1,175 @@
+package ${configs.packageName}.utils.biometrics
+
```

Comment:
were does this class come from?

Reply:
this package it's for when you need biometrics authentication in the app.